### DISCUSSION TITLE
Invisible Aircraft

### DISCUSSION THREAD

**Discussion by DEMT92** [flybywiresim/aircraft#5931] [Help] [answered]
Hey guys, been 2 days my aircraft is invisible. I tried almost everything written on internet but unfortunately, it still doesn't work. Do you have any idea ? Thanks a lot
Max

**Comment by GhostEagle68** [comment, marked as answer]
Have you tried a clean install and installing from the latest installer version?

**Reply by DEMT92** [reply]
Thank you for your answer, yes I tried twice, reinstalling either stable or expiremental version

**Reply by GhostEagle68** [reply]
Have you tried testing by removing all addons except FBW A32NX? If that works then there's a mod conflict

**Reply by DEMT92** [reply]
You suggest removing addons ingame or in the windows explorer ?

**Reply by GhostEagle68** [reply]
In the community folder

**Reply by DEMT92** [reply]
so I only had the airbus helicopter, removed it and still invisible aircraft Kappa

**Reply by GhostEagle68** [reply]
Alright so I suggest heading over to our discord and posting your issue (with steps you've done) in the support channel, that way more people can hopefully help ya 🙂

**Reply by DEMT92** [reply]
Wilco ! Thanks mate

**Reply by peteralun1** [reply]
Make sure you have no other versions installed on your system, and delete any you find.
